apps/builder/app/builder/features/sidebar-left/panels/components/components.tsx

```diff
@@ -110,7 +110,11 @@ export const TabContent = ({ publish, onSetActiveTab }: TabContentProps) => {
                             {...pressProps}
                             {...{ [dragItemAttribute]: component }}
                             label={meta.label}
-                            icon={<meta.Icon />}
+                            icon={
+                              <div
```

Comment:
maybe we could just move the rendering into the final component and pass the icon as-is as a string?